README.md

```diff
@@ -45,39 +48,39 @@ Or you can run DepTrim directly from the command line.
+| Name                     |     Type      | Description                                                                                                                                                                                                                                                                                                                                                         | 
+|:-------------------------|:-------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+| `<trimDependencies>`     | `Set<String>` | Add a list of dependencies, identified by their coordinates, to be trimmed by DepTrim. **Dependency format is:** `groupId:artifactId:version:scope`. An Empty string indicates that all the dependencies in the dependency tree of the project will be trimmed (default).                                                                                           |
+| `<ignoreDependencies>`   | `Set<String>` | Add a list of dependencies, identified by their coordinates, to be ignored by DepTrim during the analysis. Useful to override incomplete result caused by bytecode-level analysis. **Dependency format is:** `groupId:artifactId:version:scope`.                                                                                                                    |
+| `<ignoreScopes>`         | `Set<String>` | Add a list of scopes, to be ignored by DepTrim during the analysis. Useful to not analyze dependencies with scopes that are not needed at runtime. **Valid scopes are:** `compile`, `provided`, `test`, `runtime`, `system`, `import`. An Empty string indicates no scopes (default).                                                                               |
+| `<createPomTrimmed>`     |   `boolean`   | If this is `true`, DepTrim creates a trimmed version of the `pom.xml` in the root of the project, called `pom-debloated-spl.xml`, that uses the variant of the trimmed the dependencies. **Default value is:** `false`.                                                                                                                                             |
+| `<createAllPomsTrimmed>` |   `boolean`   | If this is `true`, DepTrim creates all the combinations of trimmed version of the `pom.xml` in the root of the project (i.e., `2^y`), called `pom-debloated-spl-n-x-y.xml`, where `n` is the combination number, `x` is the number of trimmed dependencies in this combination, and `y` is the total number of trimmed dependencies. **Default value is:** `false`. |
```

Comment:
Before this PR, the default behaviour was to create all combinations of pom - $2^{|debolated \\ dependencies|}$. Now you have another parameter to create the final and the most specialised POM - which one is the default behaviour? Since the default values of `createPomTrimmed` and `createAllPomsTrimmed` are `false`, it does not help me understand what will happen if I run it without any of those two.

Reply:
@algomaster99 The default behaviour of DepTrim is analyzing the dependencies and creating a folder `libs-deptrim` in the root of the project, which contains all the specialized dependencies.

The creation of specialized poms with `createPomTrimmed` and `createAllPomsTrimmed` is optional. These are additional parameters, therefore they are marked as `false` by default.

I've renamed the title of this PR to clarify this.